src/Illuminate/Foundation/Testing/Concerns/MocksApplicationServices.php

```diff
@@ -45,6 +45,44 @@ public function expectsEvents($events)
     }
 
     /**
+     * Specify a list of events that should not be fired for the given operation.
+     *
+     * These events will be mocked, so that handlers will not actually be executed.
+     *
+     * @param  array|mixed  $events
+     * @return $this
+     */
+    public function dontSeeEvents($events)
+    {
+        $events = is_array($events) ? $events : func_get_args();
+        $calledEvents = [];
+
+        $mock = Mockery::spy('Illuminate\Contracts\Events\Dispatcher');
+
+        $mock->shouldReceive('fire')->andReturnUsing(function ($called) use ($events, &$calledEvents) {
+            foreach ($events as $event) {
+                if ((is_string($called) && $called === $event) ||
+                    (is_string($called) && is_subclass_of($called, $event)) ||
+                    (is_object($called) && $called instanceof $event)) {
+                    $calledEvents[] = $event;
+                }
+            }
+        });
+
+        $this->beforeApplicationDestroyed(function () use (&$calledEvents) {
```

Comment:
don't add a `&` - it's pointless

We're not actually writing to the array, and php uses copy-on-write under the hood, so the `&` does nothing.